.env.test

```diff
@@ -0,0 +1,13 @@
+# Flask server configuration.
+FLASK_APP=app
+FLASK_ENV=development
+FLASK_SKIP_DOTENV=True
+
+# Change the following values for local development.
+
+KEY = '123'
```

Comment:
This is actually wrong, the app expects and env var called 'Key'.

But really the name should be something like TRELLO_KEY to make it clearer what it is, and to match the convention for the capitalisation of environment variables